docs/index.rst

```diff
@@ -29,7 +29,7 @@ once and reference it from anywhere you want to interface with Sentry:
 
 .. code-block:: php
 
-    $client = new \Raven\Client('___DSN___');
+    $client = new \Raven\Client('___PUBLIC_DSN___');
```

Comment:
This code is not valid anymore since `2.x` as the constructor of the class accepts a different number of parameters. You should use the client builder or inject directly the configuration object

Reply:
I will not address this here, it needs a full PR, opened #650 to track this

Reply:
I'm not really convinced that another PR is the way to go as you already edited the sections above to refer to the new classes introduced in `2.0` and so I don't see the value in not addressing this here, but if you prefer to seperate the things I won't stop you